src/openzaak/conf/includes/base.py

```diff
@@ -558,75 +179,13 @@
 #
 # DJANGO-AXES
 #
-AXES_CACHE = "axes"  # refers to CACHES setting
 AXES_FAILURE_LIMIT = 5  # Default: 3
```

Comment:
I think this setting and the two below could also be removed (as they are in OAF already with the same values)

Reply:
You're right, forgot about those! I'll remove them